Add `containerArguments` docs to Gatekeeper (#7011)
open-cluster-management/rhacm-docs: release_notes/whats_new.adoc

* To add more clarity for compliance messages in your configuration policies, you can now customize compliance messages by using the `spec.customMessage` fields. See the link:../governance/config_policy_ctrl.adoc#configuration-policy-yaml-table[Kubernetes configuration policy controller].

> * You can now view the metadata of a root policy by using the `.PolicyMetadata` hub cluster template variable. See the link:../governance/template_support_intro.adoc#template-comparison-table[Comparison of hub cluster and managed cluster templates].

Reviewer: Again, optional, but since the existing bullet list here in the what's new doc uses the sentence structure, "To < do action >, you can < do this >" consider structuring your content in a similar way: 

```suggestion
* To view the metadata of a root policy, you can use the `.PolicyMetadata` hub cluster template variable. See the link:../governance/template_support_intro.adoc#template-comparison-table[Comparison of hub cluster and managed cluster templates].
```

Reviewer: Or to make a little shorter and more imperative:

```suggestion
* To view the metadata of a root policy, use the `.PolicyMetadata` hub cluster template variable. See the link:../governance/template_support_intro.adoc#template-comparison-table[Comparison of hub cluster and managed cluster templates].
```

> * You can now use the `hubTemplateOptions.serviceAccountName` field to specify a service account to use for all hub cluster template lookups. See the link:../governance/template_support_intro.adoc#template-comparison-table[Comparison of hub cluster and managed cluster templates].

Reviewer 2: great catch !

> * You can now use the `hubTemplateOptions.serviceAccountName` field to specify a service account to use for all hub cluster template lookups. See the link:../governance/template_support_intro.adoc#template-comparison-table[Comparison of hub cluster and managed cluster templates].

Reviewer: ```suggestion
* To specify a service account to use for all hub cluster template lookups, You can now use the `hubTemplateOptions.serviceAccountName` field. See the link:../governance/template_support_intro.adoc#template-comparison-table[Comparison of hub cluster and managed cluster templates].
```
